bigquery/snippets/snippet.go

```diff
@@ -91,6 +91,60 @@ func updateDatasetAccessControl(client *bigquery.Client, datasetID string) error
 	return nil
 }
 
+func getDatasetLabels(client *bigquery.Client, w io.Writer, datasetID string) error {
```

Comment:
In Go, we generally avoid the prefix `get`. This would be `datasetLabels`.

Reply:
Side effect of naming according to region tags.  Corrected.